theme/tmp/dev-server/index.html

```diff
@@ -0,0 +1 @@
+<!DOCTYPE html><html lang="en" data-reactroot=""><head><title>landr</title><meta name="description" content="source code = website 🏠"/><meta charSet="UTF-8"/><meta name="viewport" content="width=device-width, initial-scale=1"/><link href="https://fonts.googleapis.com/css?family=Nunito&amp;display=swap" rel="stylesheet"/><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/solid.css" integrity="sha384-ioUrHig76ITq4aEJ67dHzTvqjsAP/7IzgwE7lgJcg2r7BRNGYSK0LwSmROzYtgzs" crossOrigin="anonymous"/><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/brands.css" integrity="sha384-i2PyM6FMpVnxjRPi0KW/xIS7hkeSznkllv+Hx/MtYDaHA5VcF0yL3KVlvzp8bWjQ" crossOrigin="anonymous"/><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/fontawesome.css" integrity="sha384-sri+NftO+0hcisDKgr287Y/1LVnInHJ1l+XC7+FOabmTTIK0HnE2ID+xxvJ21c5J" crossOrigin="anonymous"/></head><body><div id="root"></div></body></html>
```

Comment:
What is this theme directory?

Reply:
Probably an untracked file I had from the previous implementation - removing